rotkehlchen/exchanges/coinbase.py

```diff
@@ -285,34 +382,40 @@ def _api_query(
             options: dict[str, Any] | None = None,
             ignore_pagination: bool = False,
     ) -> list[Any]:
-        """Performs a coinbase API Query for endpoint
```

Comment:
Why did you remove the docstring here? If you want to adjust it, do that. If not then don't touch it.